src/sql/workbench/parts/query/browser/actions.ts

```diff
@@ -199,13 +200,31 @@ export class ChartDataAction extends Action {
 	public static LABEL = localize('chart', 'Chart');
 	public static ICON = 'viewChart';
 
-	constructor(@IEditorService private editorService: IEditorService) {
+	constructor(
+		@IEditorService private editorService: IEditorService,
+		@IExtensionTipsService private readonly extensionTipsService: IExtensionTipsService
+	) {
 		super(ChartDataAction.ID, ChartDataAction.LABEL, ChartDataAction.ICON);
 	}
 
 	public run(context: IGridActionContext): Promise<boolean> {
 		const activeEditor = this.editorService.activeControl as QueryEditor;
+		this.extensionTipsService.promptVisualizerExtensions();
```

Comment:
One last thought: we should not start prompting on Stable (and, likely, even on Insiders) until an update to Sandance is officially published so users can actually benefit from data visualization in the query editor scenario. For now can we get this on product quality like we do for Jupyter Books? You can do in a separate PR if needed but this is a "master checkin" requirement.

```ts
import product from 'vs/platform/product/node/product';
// To filter out just stable
if (product.quality !== 'stable') {
    // prompt
}
// To filter out stable and insiders
if (product.quality !== 'stable' && product.quality !== 'insiders') {
    // prompt
}
```